BizHawk.Client.EmuHawk/tools/GyroscopeBot/GyroscopeBot.cs

```diff
@@ -168,6 +168,80 @@ private void GyroscopeBot_Load(object sender, EventArgs e)
 
 		#region streetfighter
 
+		/*
+		 * L - High Punch
+X - Middle Punch
+Y - Low Punch
+R - High Kick
+A - Middle Kick
+B - Low Kick
+
+ 
+			 y - y
+			 h - b
+			 n - x
+			 u- a
+			 j - l
+			 m - r
+
+
+			 */
+
+		public uint get_p2_move()
+		{
+			Console.WriteLine("In get_p2_move");
+			if (is_p2_in_move())
+			{
+				/*
+				int _7e5 = _currentDomain.PeekByte(0x0007E5);
+				int _7e6 = _currentDomain.PeekByte(0x0007E6);
+				int _7e7 = _currentDomain.PeekByte(0x0007E7);
+				int _7e8 = _currentDomain.PeekByte(0x0007E8);
+				*/
+				uint val = (uint)_currentDomain.PeekUint(0x0007E5, _bigEndian);
+				return val;
+			}
+			return 0;
```

Comment:
Are you sure that 0 doesn't overlap?

Reply:
I didn't see any 0s when watching RAM, and also we do a check if we are in_move first. I don't really have a good other value for this, since the int is unsigned.